docs/templates/eiaapi_child.rst.jinja

```diff
@@ -0,0 +1,80 @@
+{% extends "data_source_parent.rst.jinja" %}
+
+{% block background %}
+EIA's API makes much of its data available through an Application Programming Interface (API).
+EIA's API is multi-facetted and contains the time-series data sets organized by the main
+energy categories:
+
+* Electricity
+* Natural gas
+* Petroleum
+* Crude oil imports
+* Coal
+* Densified biomass
+* Nuclear plant generation outages
+* Outlook of energy market/projections
+* Total energy
+* State energy data system
+* CO2 emissions
+* International energy
+{% endblock %}
+
+
+{% block availability %}
+PUDL archives all the data available within the API through the EIA's bulk API download,
+but only integrates one table of Electricity timeseries.
+{% endblock %}
+
+
+{% block respondents %}
+Respondents within the API are wide ranging because the API pulls from many different
+EIA forms. The electricity data in sourced from respondents of many of the EIA forms
+that are processed in PUDL. See the other EIA data source pages for more details.
+{% endblock %}
+
+{% block original_data %}
```

Comment:
a lot of these last two chunks i took from other places and tried to pair it down or make it less module docs more user docs but very open to suggestions